src/components/AppForm.spec.js

```diff
@@ -1,5 +1,19 @@
-describe('GameForm', () => {
-  it.todo('renders a form with an input and a button')
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import AppForm from './AppForm'
 
-  it.todo('applies the list of names to the submit callback')
+describe('AppForm', () => {
+  it('renders a form with an input and a button', () => {
+    render(<AppForm />)
+    expect(screen.getByLabelText('Favourite Wizards:')).toBeInTheDocument()
+    expect(screen.getByRole('button', { name: 'Create!' })).toBeInTheDocument()
```

Comment:
```suggestion
    expect(screen.getByLabelText(/Favourite Wizards/i)).toBeInTheDocument()
    expect(screen.getByRole('button', { name: /Create!/i })).toBeInTheDocument()
```
Using regex matchers is a little bit more robust when you add text styles to the css.

Reply:
Yea, I get why, I just don't like using something I am not really familiar with 🤷‍♀️